### ISSUE TITLE
test.wav is inaudible?

### ISSUE THREAD

**Issue by BridgetteSong** [xcmyz/FastSpeech#22]
i run the alignment.py, and  the test.wav returned by get_tacotron2_alignment_test() function is inaudible.

**Comment by 1105060120** [comment]
@BridgetteSong  i also met that problem,how did you solve that, thank you

**Comment by 1105060120** [comment]
@BridgetteSong [RECYCLE.BIN.zip](https://github.com/xcmyz/FastSpeech/files/3748747/RECYCLE.BIN.zip)
my audio file

**Comment by aguazul** [comment]
I am also having this issue - I trained to 164,000 steps but the wav file was just silent background noise.

I also tried earlier training steps (2000, 9000, 130,000) and the only one that sounded remotely like speech was 2000. By 9000 steps it was just an empty silent wav file.

I have trained using this model prior to the latest update. Did something change that is now resulting in the inaudible wav files?

**Comment by BridgetteSong** [comment]
I have not solved the problem, but my audio is better than you. 
1). I have changed the transformer architecture to tacotron architecture, and used pretrained tacotron alignments. In training, I got good result. but in reference, the audio is not clear, 
especially some last words of the sentence. sorry i have no audio demo.
2). my experiment showed this no-attention model is not effective. 
3). I think another similar architectrue is worth trying. [(AlignTTS)](https://arxiv.org/abs/2003.01950)
